src/oceans/models/guidesK5.js

```diff
@@ -14,472 +14,496 @@ const imageStyleOverrides = {
 
 const guidesK5 = [
   {
-    id: "fishvtrash-training-init1",
+    id: 'fishvtrash-training-init1',
```

Comment:
These are just linting changes, yeah?

Reply:
Yes.  For some reason my recent PR that updated these had the wrong linting.